Python/errors.c

```diff
@@ -1850,6 +1850,55 @@ PyErr_SyntaxLocationEx(const char *filename, int lineno, int col_offset)
     Py_XDECREF(fileobj);
 }
 
+/* Raises a SyntaxError and returns -1.
+ * If something goes wrong, a different exception may be raised.
+ */
+
```

Comment:
Should this newline be here?
```suggestion
```

Though, I think most editors will interpreter this as the docstring for `_PyErr_RaiseSyntaxError` anyway.